lake-manifest.json

```diff
@@ -40,5 +32,13 @@
     "opts": {},
     "name": "proofwidgets",
     "inputRev?": "v0.0.18",
+    "inherited": false}},
+  {"git":
+   {"url": "https://github.com/leanprover/std4",
+    "subDir?": null,
+    "rev": "a093d7a6ac721071d8ba3774b0318063928cd3e2",
+    "opts": {},
+    "name": "std",
+    "inputRev?": "main",
     "inherited": false}}],
```

Comment:
@tydeu, `lake` still seems to be permuting the order in `lake-manifest.json`.

Reply:
Upon review, `lake update <pkg>` will affect the order of the manifest because it pins the versions of the non-updated packages to the top (this is required to stop them from updating). Regardless, I believe manifest order is irrelevant because it is not actually used for anything in Lake (I have not double-checked this, though).

Reply:
@tydeu, it might not be used for anything in Lake, but it is unnecessarily difficult for reviewers.

Reply:
https://github.com/leanprover/lean4/issues/2664